app/src/main/java/io/gnosis/safe/ui/settings/app/GetInTouchFragment.kt

```diff
@@ -86,7 +85,12 @@ class GetInTouchFragment : BaseViewBindingFragment<FragmentGetInTouchBinding>()
             val feedback = helper.createFeedbackText(requireContext())
             val intent = Intent(Intent.ACTION_SENDTO).apply {
                 type = "text/html"
-                data = Uri.parse("mailto:${getString(R.string.email_feedback)}")
+                // gmail is ignoring intent extras
+                data = Uri.parse(
```

Comment:
will it have any effect on other mail clients (non-gmail), i.e. would they have duplicate subject/text or body if this URI is used and the intent extras - as the code is doing now?

Reply:
it is possible that some clients use intent extras and some just data uri. In worst case data will just be overridden.